### ISSUE TITLE
UI doesn't work well on small screens

### ISSUE THREAD

**Issue by naught101** [nicklan/drmr#8]
On my netbook, if there are any more than two or three rows of drums, then the bottom of the UI gets cut off, so I can't select a different drumkit. This probably wouldn't matter so much if all the kit controls were at the top, as I don't often need to change individual drums much.

It'd be great if the drums would spread out into more than 4 columns when the window was resized too, as this would allow easier access to most drums.

**Comment by naught101** [comment]
Also, if the sample light/preview button was at the top of the drum panel, next to the drum name, then you'd save a bit of vertical height per row (although the name might get cut a bit).

**Comment by naught101** [comment]
whoops

**Comment by nicklan** [comment]
yeah, the interface certainly needs some revision.  i'll look at getting some of the minor updates you've suggested in soon, as i agree they would be useful.  when i finally find time to add in kit editing i think the ui will need some significant re-design, but i'll cross that bridge when i come to it :)

**Comment by ohac** [comment]
FYI: I use this ad-hoc patch.
ohac/drmr@58d6e501c01e6b25e7b37ea09234d1a7f9403daa
